scripts/install.js

```diff
@@ -65,6 +66,39 @@ function createProgressBar(name, total) {
   return { tick: () => {} };
 }
 
+function copyFileSync(src, dest) {
+  const data = fs.readFileSync(src);
+  fs.writeFileSync(dest, data);
+}
```

Comment:
This copy function does not preserve file mode, and as a result, the output file does not have the execution bits set. Because of that we get the following CI failure: https://travis-ci.com/getsentry/sentry-cli/jobs/158100653

Additionally, `sentry-cli` binaries are (sadly) not that small at the moment (~12-20 MB), so it would be a bit nicer not to read the entire file into memory and use read/write streams instead.

Reply:
oh no 🤦.

ok just to use this?
https://www.npmjs.com/package/fs-copy-file-sync

Reply:
Scrap that. Went with `fs-extra` because we also need to use `mkdirp` or similar.